### PR TITLE
35 improve doc strings

### PR DESCRIPTION
**Pull request by dtabell**
In addition to clarifying many of the docstrings—especially the direction in
which the conversion factor operates—I have extended the dictionary of SI
prefixes, fixed a bug (typo) in the nano prefix, improved the output formatting
of printunits(), and reformatted some of the code.

I think we can close this issue for now. Further work on the docstrings can
happen as we transition to the use of `Unitful`.

### CODE REVIEW

**Comment by DavidSagan** [comment]
Did a quick look and things look good. @lllx125 @rot4te I have assigned you to review this. There needs to be a discussion as to what the procedure is in the future.

**Comment by dtabell** [comment]
I amended the Unit dictionary so that the tests now all pass.
But the switch to `Unitful` may necessitate a complete rewrite.

File: src/SetUnits.jl [hunk 1]
```diff
@@ -1,47 +1,57 @@
-
 # include("PhysicalConstants.jl")
```
Comment (rot4te) [inline src/SetUnits.jl:1]: includes should be suppressed in the final version, as everything is included from AtomicAndPhysicalConstants.jl
if they're left in subsidiary files, I think they throw errors.

Comment (rot4te) [inline src/SetUnits.jl:1]: Includes should be suppressed in the release version, as repeated includes cause errors and everything is included directly in AtomicAndPhysicalConstants.jl

**Pull request closed without merging**